### ISSUE TITLE
Zooming doesn't work in 8.1.5

### ISSUE THREAD

**Issue by Shushuda** [Mpstark/DynamicCam#16]
Hello! 
Today's patch broke zooming. Changing situations (going indoors, outdoors, into a city, mounting, etc) doesn't zoom in nor out, no matter the setting used. The zoom always stays the same. 
Shoulder offset still seems to work as intended tho. Only zooming seems to be broken.

**Comment by LudiusMaximus** [comment]
Hm, I just tried and it seems to work for me.

**Comment by Shushuda** [comment]
Hmm, that seems pretty weird, I still cannot get it to work. I will try to see if I can pinpoint the cause on my side.

**Comment by LudiusMaximus** [comment]
Maybe you can use the "export profile" option and post the string here? Then I could try your settings.

**Comment by Shushuda** [comment]
`dSKZDaGiKOrrk6uKsnlrXTOsk7sf6xiHedtehtvAziLNPkmnQextuzBQQ8nvGXjfCovqRJkPsVdj4EQQYbvvAHIspuv0efvDrsHnsf(OurzKivCsPsRePsVejuzMiH4Msr7uL(jsvdLk6OiHKwQQQ6PkMkuDvQKSvPcFfjuATujv9wKqQ7Iek2l4VqXGL0HPSyPqpwOjlvDzuBwvXNLsJgkDAsETur1Sjv3gXUH8BLgUiz5u1ZfmDIRJK2ovQVlvKXtk58IuRxf9EQKkUpsOQ9lXWlGdtk)(rPZaom98hJQU80T5daomchemrJWElFAKrurNkjInRfyhRBNwpCtoM0amchemrJWEJoS13CDZegj)BOwEIModxxGr4GGjAe2B5tJmIk6ujrSdX8ItL7vEsv2pFYRccCtGr4GGjAe2B5tJmIk6ujrSzTa7yD706ZVDcnrtNHBcmchemrJWElFAKrurNkjIfUVWiCqWenc7n6DZ(b7svVN9W994lmchemrJWElFAKrurNkjInRfyhRBNwVd90kulCto6YpyIgH9MRFxYKI5E4s7qqGXn7dQfbxAW8r5dY)urqJW4zlkk3SRFxcGdJ133ff9mHrs4RENS0b4MaJ133ff9mHrsaU5GHyA563LGRimCPbcm98hJQU0b7dQfb3xyckHQjQffaCysE8aWHHDZija4WnhmACFsVRU4A8pMbNENGat0iS3(gf1IcaomN7sCOe8TTvNVgHtPBcdzH5Cxsttl4slbeqGjH2dahg2nJKaGd3CWOX9j9U6IRX)ygC6DccmiJWWKNkQLxuPQ5ZL0Ojzk1SBiOnmXf1Re1IGrFJMEPQcDrbXqrotPQqbXqrgpDkUu)Ru7g6I6IcIHISMAtHs1ZcvpQu1uf6IcIHICPAO(svHcIHImE6uCP(3FLkLPmVKMsTlvd1xQUcPKVF(WrrHAFUKgnrtktPBPzpLAxQgQVu7g0afxQiH9Dro9BRNdbn1McWenc7TVrrTOaGdZ5UehkbFBB15Rr4u6MWqwyo3LmSikulmFucmN7sAAAbxAjGacmjj)aCyIgH923OOwuaWH5CxIdL4Qa)KQ)BOPcbzH5Cxsttl4EamN7sAQqW1fyeVHckrTiCfHH77XCWCUlPdtNkd3ChZbZ5UehkbFBB15Rr4u6MWqwyo3L4qjDrrptyKeGBYXCWCUlXHsEUirHwQbilmN7sgwefQfMpkbeyIgH92ZomFaWHr4GGjAe2B5tJmIk6ujrSzTa7yD706HBYXCWiCqWenc7T8PrgrfDQKiw4(cJWbbt0iS3OdB9nx3mHrYiEg1kyH7lmchemrJWElFAKrurNkjInRfyhRBNwF(TtOjA6mCto(4fgHdcMOryVrVB2pyxQ69ShUVhPbJWbbt0iS3YNgzev0PsIyhI5fNk3R8KQSF(Kxfe4(MCmhiGaZJeAaomrJWE7zhMpa4WiCqWenc7T8PrgrfDQKiw4(ccmrJWE7Buulka4WCUlzyruOwycSaZ5UKomDQmCpaMZDjouc(22QZxJWP0nHHSWCUlPPPfCPLaciWKKKdWHjAe2Bp7W8bahgHdcMOryVrh26BUUzcJKr8mQvWc3KJ5Gr4GGjAe2B07M9d2LQEp7H77rAGat0iS3(gf1IcaomI3qbLOweUIWW994lmN7sAAAbxAjWCUlPdtNkd3hhZbZ5UKHfrHAHrHG5CxIdLGVTT681iCkDtyiliGatscnahMOryV9nkQffaCyeVHckrTiCfHH77XKCWCUlPPPfCPLaZ5UKomDQmCFCmhmN7sgwefQfMalWCUlXHsW32wD(AeoLUjmKfeyIgH92ZomFaWHr4GGjAe2B5tJmIk6ujrSzTa7yD706HBYX3xyeoiyIgH9g9Uz)GDPQ3ZE4(EKgmchemrJWElFAKrurNkjIfUVWiCqWenc7T8PrgrfDQKi2SwGDSUDA953oHMOPZWnbeqGj5XlGddJSg15EOryIgH923OOwuaWH5CxIdLGVTT681iCkDtyilmN7sAAAbxAjGacmjpsaCyyK1Oo3dnct0iS3(gf1IcaomN7sCOe8TTvNVgHtPBcdzH5Cxsttl4slbeqG5nHgGdt0iS3(gf1IcaomN7sCOe8TTvNVgHtPBcdzH5Cxsttl4slbeyIgH92ZomFaWHr4GGjAe2B07M9d2LQEp7bhU2b0sYDqUdDj5Xbhs7GdObFCTC)Gr4GGjAe2B0HT(MRBMWizepJAfSWnbgHdcMOryVLpnYiQOtLeXc3eqabMKCjaommYAuN7HgHjAe2BFJIArbahMZDjouc(22QZxJWP0nHHSWCUlPPPfCPLaciWKqlbWHbzegM8urT8IGjUOELOwem6B00lvvOlkigkYzkvfkigkY4PtXL6FLA3qxuxuqmuK1uBkuQEwO6rLQMQqxuqmuKlvd1xQkuqmuKXtNIl1)(RuPmL5L0uQnfGjAe2BFJIArbahMZDjdlIc1ctGfyo3L0HPtLH7Boyo3L4qj4BBRoFncNs3egYcZ5UKMMwWLwciGaZJKxahggznQZ9qJWenc7TVrrTOaGdZ5UKHfrHAHrHG5CxshMovgUUaZ5UehkbFBB15Rr4u6MWqwyo3L000cU0sabeys(La4WWiRrDUhAeMOryV9nkQffaCyo3L4qj4BBRoFncNs3egYcZ5UKMMwWLwciGatYdxaCyy3mscaoCZbJg3N07QlUg)JzWP3jiWWiRrDUhAeMOryV9nkQffaCyo3L4qj4BBRoFncNs3egYcZ5UKMMwWLwciGatcTxahgKryyYtf1YlQu1KEQs)U(G2WexuVsulcg9nA6LQk0ffedf5mLQcfedfz80P4s9VsTBOlQlkigkYAQnfkvplu9OsvtvOlkigkYLQH6lvfkigkY4PtXL6F)vQuMY8sAk1UunuFP2nqpvPFxFqtTPamrJWE7Buulka4WCUlzyruOwycSaZ5UKomDQmCFZbZ5UehkbFBB15Rr4u6MWqwyo3L000cU0sabeyssCbWHjAe2Bp7W8bahgHdcMOryVLpnYiQOtLeXM1cSJ1TtRhUjhD5hmchemrJWElFAKrurNkjIfUVWiCqWenc7n6WwFZ1ntyKmINrTcw4MCSbyeoiyIgH9w(0iJOIovseBwlWow3oT(8BNqt00z4MC83pyeoiyIgH9w(0iJOIovse7qmV4u5ELNuL9ZN8QGa3eyeoiyIgH9g9Uz)GDPQ3ZE4(cJWbbt0iS3YNgzev0PsIyZAb2X62P17qpTc1c3KJ)(bcmrJWE7Buulka4Wikc3n000cAeMZDjDy6uz4(dMZDjdlIc1ctGfyo3L4qj4BBRoFncNs3egYcZ5UKMkeC)bJ4nuqjQfHRimCFpMKdMZDjnnTG7BciGatsYlGdt0iS3(gf1IcaomI3qbLOweUIWW9fMZDjnnTG7pyo3L0HPtLH7pyo3LmSikulmbwG5CxIdLGVTT681iCkDtyiliWenc7TNDy(aGdJWbbt0iS3OdB9nx3mHrYiEg1kyHBcmchemrJWEJE3SFWUu17zpCFpsdeqGjHMlaomSBgjbahU5GrJ7t6D1fxJ)Xm407eeyqgHHjpvulVOsvZNlPrtYuQ4ElGvByIlQxjQfbJ(gn9svf6IcIHICMsvHcIHImE6uCP(xP2n0f1ffedfzn1McLQNfQEuPQPk0ffedf5s1q9LQcfedfz80P4s9V)kvktzEjnLAxQgQVuDfsjF)8HJIc1(CjnAIMuMs3sZEk1UunuFPIwPu7g0afxQiH9Dro9BRNdbn1McWenc7TVrrTOaGdZ5UKMkeCFtG5CxYWIOqTW8rjWCUlXHsW32wD(AeoLUjmKfeqGjj3lGddJSg15EOryIgH923OOwuaWH5CxIdLGVTT681iCkDtyilmN7sAAAbxAjGacmj)EbCyyK1Oo3dnct0iS3(gf1IcaomN7sCOe8TTvNVgHtPBcdzH5Cxsttl4slbeqG5rscGdt0iS3E2H5daomchemrJWEd38TS4yJudDrc7TOObYAuNH7lmchemrJWEJoS13CDZegjJ4zuRGfUVWiCqWenc7T8PrgrfDQKi2SwGDSUDA953oHMOPZW9fgHdcMOryVLpnYiQOtLeXM1cSJ1TtRhUVWiCqWenc7T8PrgrfDQKiw4(cJWbbt0iS3YNgzev0PsIyhI5fNk3R8KQSF(Kxfe4MaJWbbt0iS3WnFllo2i1GgiMKwdK1Ood3xyeoiyIgH9g9Uz)GDPQ3ZE46AVhBagHdcMOryVLpnYiQOtLeXM1cSJ1TtR3HEAfQfUjh)9deyIlQxjQfbJ(gn9snGD7eQu)Ru)y6bMcf6(z9LANNXDPQqLAkt5dAkyvHJF8gHdAxQ9BHULAPwQLQ6tPQjMoJI2nftPAO(sftNrr7MIX1TByWUDcPP2AxQcwgvOBPwQLAPwQLAPwQbSBNqL6FLQ4PYuOq3sTul1sLr9f6YO(cD9Sq1JkvxHuIgAPcsqtkrPIuQDPAO(s1viL0n4kKs0KsuQiLzkvkfZ3YcLAxQgQVudy3oHOamSwCKQW0J6Iucmcwv44hVr4qP(xP66qzNzEVkSONwklQeLBoVkKs3s7WBeMYmLkLznuN7WBeMYmLkLnzFeRHehEJWuMPuPSZ2qqLYH3imLzkvkFQBHaU3ui27WBeMYmLkLuuPYbXH3imLzO4HjAe2BFJIArbahMZDjouIRc8tQ(VHMkeKfMZDjnnTGlTgoMdMZDjnvi4MCmhmN7sCOe8TTvNVgHtPBcdzHXVIjCwl25qwy8RycdzH5CxIdL0ff9mHrsaUjhZbZ5UehkX5guIArW9Weyo3L0HPtLH7JJ5GjT1P8CRN5aCDnxsGXVIj8WIOqTW0ZTEMdWCUlzyruOwyuiyeVHckrTiCfHHBYXCGad7MrsaWH77lmn14)N057fhZtkc9Jg5H77dyW))B3Rg)J5jfH(rJ8WTbyY(9sX54)Vn1ad9o14QrE4(Mattn()jD(EXXm0HEkw4(sdg8))29QX)yg6qpflCFDbMHoDD2uJloMN))1bUg)JHENAC1ipCpeMSFVuCo()BtnW8KIq)OrE4(MdMHoDD2uJloMN))1bUg)J5jfH(rJ8Gacm0ssaCyyT4ivHPhPbsypm6B00lvmzkvmzkvmzkvmzk1GyEbxr4mLkJ6XveUu)RuDfsjpTGOqTDrFwnPmLULM9uQnfk0vWQchfVHckrTiCfHl1)kvn1Kr94kcxQUwPgeZl4kcRnfL3KKeTlvxRupslhfGjAe2Bp7W8bahgHdcMOryVLpnYiQOtLeXc3eyeoiyIgH9gDyRV56MjmsgXZOwblCtGr4GGjAe2B07M9d2LQEp7HBciWWAjEla4WGv1ZUQlKfeyyT4ivHPh1fPeyeSQWXqkwxpuQ)vQUoVp(LlPHmL6Bdh2WJgYuQ5CXLKFzk1dD5G3dZuQ5Y9ltP((RbxA4GmL67VgC5WdZuQVh8sl3JmLAUKdp4GmLAd)omjtPsJgTFhMltP(EWlTCpYuQ0O5sd)omtPM7XRljzk1h5(LltP(GgTdEZuQVh(4GChMPuFK7hTmL6JC)oitP(4XVdtYuQpYDWBUmL6dA0oGwMs1LdFKRHmL67b)O5sdzk1h5(9MPuD5Wh5omtP(i3VFzk1gA4Hl0YuQn0WdxCjtP(i3Vhzk13h0(r7GmL67dA)O9MPuFp4hnxOLPuFpKwsd5YuQVhslPbxYuQ0E)L8(JINcWexuVsulcMpRVu78mUlvfQutzkFqtbRkCmKI11dAxQ9BPwOBPwQLAPQ(uQA6kKsEAbrHA7I(SAszkDln7Pu7s9V)k1oJLjfRR3f9z10T2AxQcwgvQf6wQLAPwQLAPwQLQNfQEuPkEQmfk0Tul1sTuzuFHUmQVqxplu9Os9JPhykat0iS3(gf1IcaomN7sAAAbxAjWCUlzyruOwyuiyeVHckrTiCfHH7Bcm(vmHhwefQfMEU1ZCaMZDjouc(22QZxJWP0nHHSWK26uEU1ZCaUp(LaJFftyilmIIWDdnnTGgH5CxshMovgUUaciW8MKa4Wenc7TNDy(aGdJWbbt0iS3YNgzev0PsIyZAb2X62P1d3KJpGr4GGjAe2B5tJmIk6ujrSW9fgHdcMOryVrh26BUUzcJKr8mQvWc3KJUaJWbbt0iS3YNgzev0PsIyZAb2X62P1NF7eAIMod3eyeoiyIgH9g9Uz)GDPQ3ZE4MdgHdcMOryVLpnYiQOtLeXoeZlovUx5jvz)8jVkiWTHJ5abM4I6vIArW4zHQhvQDdnxQiH71uBkat0iS3(gf1IcaomN7sCOe8TTvNVgHtPBcdzHr8gkOe1IWvegUVWCUlzyruOwywQcmN7sAAAbxAjGacm0sEbCyIgH92ZomFaWHr4GGjAe2B5tJmIk6ujrSWnbgHdcMOryVrh26BUUzcJKr8mQvWc3eyeoiyIgH9g9Uz)GDPQ3ZE4MacmXf1Re1IG5Z6l1opJ7svHk1uMYh0uWQchBK6NpbTl1(Tul0Tul1sTuvFkvnDfsjzP(5JMuMs3sZEkZuQDgltkwxVl6ZQPBT1w7svWYOcDl1sTul1sTul1s1ZcvpQufpvMcf6wQLAPwQmQVqxg1xORNfQEuP(X0dmfGjAe2BFJIArbahMZDjouc(22QZxJWP0nHHSWCUlPPPfCPLaciW8M8c4WWAjEla4WGv1ZUQlKfeyIgH923OOwuaWHXVIj8KI5EWHlh0qUKdC5xIljn8sljb(4AU0amN7sAAAbxAjWCUlXHsW32wD(AeoLUjmKfg)kMWqwyo3LmSikulmbwG5CxshMovgUV0abMOryV9SdZhaCyeoiyIgH9gDyRV56MjmsgXZOwblCPbJWbbt0iS3O3n7hSlv9E2d3hhBaeqabgR3THICpKfeyK0PyyY)eJZ)P3rxkIgWGmcddUpTc1wQ4yvO2amgvb76HzWsnGLAVbcaa`

**Comment by Shushuda** [comment]
I've fiddled with it a bit and I've noticed that my old profiles just refuse to zoom. Exporting a string and then importing it still gives me a non-working profile, no matter which one (I got a few) I try. However, creating an empty profile and setting it *exactly* the same as the old profile seems to make it work. So I guess I just need to manually set everything again on new profiles. I have no idea why it's so, but at least now it seems to work.
